app/test/prepare_proposal_test.go

```diff
@@ -80,42 +100,14 @@ func TestPrepareProposalBlobSorting(t *testing.T) {
 		res := testApp.PrepareProposal(tt.input)
 		assert.Equal(t, tt.expectedBlobs, res.BlockData.Blobs)
 		assert.Equal(t, tt.expectedTxs, len(res.BlockData.Txs))
-
-		// verify the signatures of the prepared txs
-		sdata, err := signer.GetSignerData()
-		require.NoError(t, err)
-
-		dec := encoding.IndexWrapperDecoder(encCfg.TxConfig.TxDecoder())
-		for _, tx := range res.BlockData.Txs {
```

Comment:
this portion of this test was leftover from when we were malleating txs for multiple square sizes, so we can remove. It no longer works since this portion of the test only works when we don't need access to state.